apps/admin-portal/src/pages/claim-dialects.tsx

```diff
@@ -101,46 +103,52 @@ export const ClaimDialectsPage = (): React.ReactElement => {
     }, [sortBy, sortOrder]);
 
     /**
-     * This slices a portion of the list to display
-     * @param {ClaimDialect[]} list
-     * @param {number} limit 
-     * @param {number} offset 
-     * @return {ClaimDialect[]} Paginated List
+     * This slices a portion of the list to display.
+     *
+     * @param {ClaimDialect[]} list.
+     * @param {number} limit.
+     * @param {number} offset.
+     *
+     * @return {ClaimDialect[]} Paginated List.
      */
     const paginate = (list: ClaimDialect[], limit: number, offset: number): ClaimDialect[] => {
         return list?.slice(offset, offset + limit);
     };
 
     /**
-     * Handles change in the number of items to show
-     * @param {React.MouseEvent<HTMLAnchorElement>} event
-     * @param {data} data
+     * Handles change in the number of items to show.
+     *
+     * @param {React.MouseEvent<HTMLAnchorElement>} event.
+     * @param {data} data.
      */
     const handleItemsPerPageDropdownChange = (event: React.MouseEvent<HTMLAnchorElement>, data: DropdownProps) => {
```

Comment:
Fix other places as well.

```suggestion
    const handleItemsPerPageDropdownChange = (event: React.MouseEvent<HTMLAnchorElement>, data: DropdownProps): void => {
```